### PR TITLE
<RangeSlider />

### PR DESCRIPTION
**Pull request by fAns1k**
Implemented RangeSlider component:

rangeslider_ios
rangeslider_android

### CODE REVIEW

**Comment by itsmepetrov** [comment]
@fAns1k please update tests

**Comment by itsmepetrov** [comment]
Blocked by #4

File: src/index.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ export { default as StarRating } from './StarRating'
 export { default as Tags } from './Tags'
 export { default as Breadcrumbs } from './Breadcrumbs'
 export { default as Carousel } from './Carousel'
+export { default as RangeSlider } from './RangeSlider'
```
Comment (isnifer) [inline src/index.js]: Please add also 
```js
export { default as Label } from './Label'
export { default as LabelRating } from './LabelRating'
```

File: __tests__/e2e/08_RangeSlider.test.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    t.pass('first marker should be visible')
+
+    await driver.waitForVisible(secondMarkerId, 10000)
+    t.pass('second marker should be visible')
+
+    await driver.waitForVisible(minValueId, 10000)
+    t.pass('min value should be visible')
+
+    const minValueText = await driver.getText(minValueId)
+    t.equal(minValueText, '10', 'min value should be 2')
```
Comment (isnifer) [inline __tests__/e2e/08_RangeSlider.test.js]: should be 10

**Pull request merged**